ControlExtensions/src/test/java/demoqa/tests/DropDownListTest.java

```diff
@@ -0,0 +1,43 @@
+package demoqa.tests;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import demoqa.pages.DropDownListPage;
+import framework.TestBase;
+
+public class DropDownListTest extends TestBase {
+	@Test
+	public void canSelectDropDownListValue() {
+		String expectedResultText = "Indigo";
+
+		String actualResultText = new DropDownListPage(this.getDriver()).navigate().selectOption(expectedResultText)
```

Comment:
Put each step on a separate line

new DropDownListPage(this.getDriver())
.navigate()
.selectOption(expectedResultText)